compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -698,6 +698,9 @@ static void addLowerToLLVMPasses(OpPassManager &passManager,
 
   if (enableAArch64SME) {
     // (Arith, Vector) -> ArmSME
```

Comment:
This comment should be moved. Also, it would be good to add a note explaining what the legalization pass does :)